client/app/components/composites/ListingCardPanel/ListingCardPanel.js

```diff
@@ -0,0 +1,38 @@
+import React, { Children, Component, PropTypes } from 'react';
+import { div } from 'r-dom';
+import classNames from 'classnames';
+import css from './ListingCardPanel.css';
+
+
+class ListingCardPanel extends Component {
+
+  shouldComponentUpdate(nextProps) {
+    return this.props.children !== nextProps.children;
+  }
+
+  render() {
+    const childrenWithColumnStyle = Children.map(
+      this.props.children,
+      (child) => React.cloneElement(child, {
```

Comment:
Does this mean we have two copies of the element in memory? Just wondering if there was a better way to pass elements as children.